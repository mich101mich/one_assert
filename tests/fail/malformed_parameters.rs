fn main() {
    one_assert::assert!(1 == 2, "{}");
    one_assert::assert!(1 == 2 "{}");
    one_assert::assert!(1 == 2, , "{}");
    one_assert::assert!(1 == 2, "{}", 1, 2);
}
