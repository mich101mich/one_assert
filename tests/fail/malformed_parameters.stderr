error: 1 positional argument in format string, but no arguments were given
 --> tests/fail/malformed_parameters.rs:2:34
  |
2 |     one_assert::assert!(1 == 2, "{}");
  |                                  ^^

error: condition has to be followed by a comma, if a message is provided
 --> tests/fail/malformed_parameters.rs:3:32
  |
3 |     one_assert::assert!(1 == 2 "{}");
  |                                ^^^^

error: expected expression, found `,`
 --> tests/fail/malformed_parameters.rs:4:33
  |
4 |     one_assert::assert!(1 == 2, , "{}");
  |                                 ^ expected expression

error: argument never used
 --> tests/fail/malformed_parameters.rs:5:42
  |
5 |     one_assert::assert!(1 == 2, "{}", 1, 2);
  |                                 ----     ^ argument never used
  |                                 |
  |                                 formatting specifier missing
